pandas/core/window/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pandas.core.window.ewm import EWM  # noqa:F401
+from pandas.core.window.expanding import Expanding  # noqa:F401
```

Comment:
I would add ExpandingGroupby & RollingGroupby here as well; we want to export the 'public' interface to the rest of pandas, so IOW, we don't have *any* where that needs to reach into pandas.core.window.*, then just import from pandas.core.window (this is all for inside of pandas)